src/redux/actions/order.js

```diff
@@ -1,17 +1,22 @@
+import _ from 'lodash'
 import ACTIONS from '../actionTypes'
+import web3Instance from '../../singletons/web3/web3'
 
-const add = () => {
+const add = (itemId, sellingQuantity, unitPrice) => {
```

Comment:
`addItem({ itemId, quantity })` 

1. Rename

2. No need to pass in the price. Make a selector for item price and use it here:
```js
const price = selector.items.price(getState(), { itemId })
```

3. It's best to wrap function args in an object when there is more than one
```js
addItem(itemId, quantity)
// vs
addItem({ itemId, quantity })
```

This way when you call the function, parameter order does not metter and params are named:
```js
addItem({ quantity: 5, itemId })
```